src/main/java/org/broadinstitute/hellbender/utils/IndexUtils.java

```diff
@@ -26,16 +31,17 @@ private IndexUtils(){}
      * Load a Tribble .idx index from disk, checking for out of date indexes and old versions
      * @return an Index, or null if we're unable to load
      */
-    public static Index loadTribbleIndex(final File featureFile) {
+    public static Index loadTribbleIndex(final Path featureFile) {
```

Comment:
What was the impetus for these changes?

Reply:
This is necessary to get the sequence dictionary out of the tabix index over NIO because the results I was working with were from an older version.  I've since added the sequence dictionary to the header, but this bug should be fixed.